RecoLocalCalo/HGCalRecAlgos/src/RecHitTools.cc

```diff
@@ -150,6 +150,10 @@ GlobalPoint RecHitTools::getPositionLayer(int layer, bool nose) const {
     }
   } else {
     const HGCalDDDConstants* ddd = get_ddd(geom_, geometryType_, fhOffset_, lay);
+    if (geometryType_ == 1) {
+      if (lay > fhOffset_)
```

Comment:
Shouldn't this be ">="?  (Just guessing...)

Reply:
No, the `fhOffset_` is the last layer of EE (=28), and only for the first layer of the hadronic calorimeter (29) the offset correction (29-28=1) be used, since the layer numbering starts with 1.